.prettierrc

```diff
@@ -3,5 +3,6 @@
   "singleQuote": true,
   "trailingComma": "none",
   "bracketSpacing": false,
-  "printWidth": 100
+  "printWidth": 100,
+  "arrowParens": "always"
```

Comment:
I saw you added parentheses, that is removed by Prettier that runs on pre-commit.
So I changed this configuration to prevent removing parentheses from Prettier.
https://github.com/cssinjs/jss/commit/c60f3955646ffaa95c1121f85ce7942b1873126d

I found lint you committed is more likely written by Prettier v2 rather than v1 installed by devDependency.
When I accidentally run prettier installed globally in my machine and the result was similar to https://github.com/cssinjs/jss/commit/c60f3955646ffaa95c1121f85ce7942b1873126d .
`arrowParens` is `"always"` by default in Prettier v2.

Reply:
lets upgrade prettier

Reply:
upgraded 👍